src/schema/v2/filterArtworksConnection.ts

```diff
@@ -273,12 +273,27 @@ export const FilterArtworksFields = () => {
       type: new GraphQLList(Artist.type),
       description:
         "Returns a list of merchandisable artists sorted by merch score.",
-      resolve: ({ aggregations }, _options, { artistsLoader }) => {
+      args: {
+        size: {
+          type: GraphQLInt,
+          description: "The number of artists to return",
```

Comment:
I've left this as an optional arg, with no default value, so as not to break existing clients. (This might mean that overfetching is still happening out in the wild, but new clients can opt in this arg.)

Reply:
Was thinking about https://github.com/artsy/metaphysics/pull/2130/files#r369237215 and I would be 👍 a reasonable default.

Reply:
Yeah, I'm not opposed to that. I will follow-up to see what product implications there could be for this 👍

Reply:
I've yet to see more than 12 artists appear in the Featured Artists list, and I think that conforms to this upstream limit:
https://github.com/artsy/gravity/blob/0d53400113c5236bda3bee1e2822c497b88f64f9/app/models/search/queries/artwork_filtered_query.rb#L34

So I'll go with that as the default (though finding this^ makes me wonder just how much of a problem we are actually solving with this PR 😬). 

But best not to overfetch if it can be avoided, so I'll proceed with this.

Reply:
Oh, nice sleuthing!